src/consensus/aft/raft.h

```diff
@@ -1138,7 +1134,7 @@ namespace aft
       {
         become_aware_of_new_term(r.term);
       }
-      else if (state->current_view < r.term)
+      if (state->current_view < r.term)
```

Comment:
```suggestion
      else if (state->current_view < r.term)
```

Speculative fix - this seems like a semantic change that could be causing the test failure?